py/server/deephaven/experimental/iceberg.py

```diff
@@ -243,6 +340,95 @@ def table(self, instructions: Optional[IcebergInstructions] = None, snapshot_id:
+
+        for table in tables:
+            builder.addDhTables(table.j_table)
+
+        for partition_path in partition_paths:
+            builder.addPartitionPaths(partition_path)
+
+        if instructions is not None:
+            builder.instructions(instructions.j_object)
+
+        return self.j_object.append(builder.build())
+
+    def overwrite(self,
+                  table_identifier: str,
+                  tables: List[Table],
+                  partition_paths: Optional[List[str]] = None,
+                  instructions: Optional[IcebergParquetWriteInstructions] = None):
+        """
+        Overwrite the existing Iceberg table with the provided Deephaven tables in a single snapshot. This will
+        overwrite the schema of the existing table to match the provided Deephaven table if they do not match.
+        Overwriting a table while racing with other writers can lead to failure/undefined results.
+
+        Args:
+            table_identifier (str): the identifier string for iceberg table to overwrite.
+            tables (List[Table]): the tables to overwrite.
+            partition_paths (Optional[List[str]]): the partitioning path at which data would be overwritten, for example,
+                "year=2021/month=01". If omitted, we will try to overwrite data to the table without partitioning.
+            instructions (Optional[IcebergParquetWriteInstructions]): the instructions for customizations while writing.
+        """
+        builder = _JIcebergOverwrite.builder().tableIdentifier(table_identifier)
+
+        for table in tables:
+            builder.addDhTables(table.j_table)
+
+        for partition_path in partition_paths:
+            builder.addPartitionPaths(partition_path)
+
+        if instructions is not None:
+            builder.instructions(instructions.j_object)
+
+        return self.j_object.overwrite(builder.build())
+
+    def write_data_file(self,
+                        table_identifier: str,
+                        tables: List[Table],
+                        partition_paths: Optional[List[str]] = None,
+                        instructions: Optional[IcebergParquetWriteInstructions] = None):
+        """
+        Writes data from Deephaven tables to an Iceberg table without creating a new snapshot. This method returns a list
+        of data files that were written. Users can use this list to create a transaction/snapshot if needed.
+
+        Args:
+            table_identifier (str): the identifier string for iceberg table to write to.
+            tables (List[Table]): the tables to write.
+            partition_paths (Optional[List[str]]): the partitioning path at which data would be written, for example,
+                "year=2021/month=01". If omitted, we will try to write data to the table without partitioning.
+            instructions (Optional[IcebergParquetWriteInstructions]): the instructions for customizations while writing.
+        """
```

Comment:
All above cases that are missing the return type hint are also missing docs on the return value